aapcs64/aapcs64.rst

```diff
@@ -892,6 +892,9 @@ thread-local storage on platforms where multi-threaded code is
 supported.  The exact location of such information is platform
 specific.
 
+The FPMR is a system register that controls behaviors of the FP8 instructions.
```

Comment:
is it worth to tag this section as alpha for now ?

Reply:
ok